spec/requests/users_spec.rb

```diff
@@ -29,20 +36,16 @@
     end
   end
 
-  describe "GET /edit" do
-    it "renders a successful response" do
-      user = FactoryBot.create(:user)
-      get edit_user_url(user)
-      expect(response).to be_successful
-    end
-  end
-
   describe "POST /create" do
     context "with valid parameters" do
-      it "creates a new user" do
+      it "creates a new user with correct attributes and associations" do
         expect {
           post users_url, params: { user: valid_attributes }
         }.to change(User, :count).by(1)
+        expect(User.last.email).to eq("new_name@example.com")
```

Comment:
Since `User.last` is the same here, you can store it in local var reusable for multiple assertions